README.md

```diff
@@ -1,22 +1,64 @@
 ![](https://github.com/ISISComputingGroup/saluki/blob/main/resources/logo.png)
 
-Serialise/deserialise flatbuffers blobs from kafka. 
-This currently deserialises https://github.com/ess-dmsc/python-streaming-data-types, but I am working to make it agnostic. Python bindings for the respective schema will need to be generated. 
+ISIS-specific Kafka tools.
+Deserialises [the ESS flatbuffers blobs](https://github.com/ess-dmsc/python-streaming-data-types) from Kafka. 
+
+Also allows replaying data in a topic. 
 
 # Usage
+
+To run the latest version, install [uv](https://docs.astral.sh/uv/getting-started/installation/) and use `uvx saluki <args>`.
+
+alternatively you can `pip install saluki` and run it from a `venv`. 
+
 See `saluki --help` for all options. 
 
-## Listen to a topic for updates
+## `listen` - Listen to a topic for updates
 `saluki listen mybroker:9092/mytopic` - This will listen for updates for `mytopic` on `mybroker`. 
 
-## Consume from a topic
+### Filter to specific schemas
+
+`saluki listen mybroker:9092/mytopic -f f144 -f f142` - This will listen for updates but ignore messages with schema IDs of `f142` or `f144`
+
+## `consume`- Consume from a topic
 `saluki consume mybroker:9092/mytopic -p 1 -o 123456 -m 10` - This will print 9 messages before (and inclusively the offset specified) offset `123456` of `mytopic` on `mybroker`, in partition 1.
 
 Use the `-g` flag to go the other way, ie. in the above example to consume the 9 messages _after_ offset 123456
 
-# Install 
-`pip install saluki`
+You can also filter out messages to specific schema(s) with the `-f` flag, like the example above for `listen`.
```

Comment:
Maybe this should also support timestamps - offsets are not very user friendly...